Loadsys/Sniffs/NamingConventions/ValidPrivateProtectedFunctionNameSniff.php

```diff
@@ -98,7 +98,7 @@ protected function processTokenWithinScope(PHP_CodeSniffer_File $phpcsFile, $sta
 		if ($isPublic === true) {
 			if ($methodName[0] === '_') {
 				$error = 'Public method name "%s" must not be prefixed with underscore';
-				$phpcsFile->addError($error, $stackPtr, 'PublicWithUnderscore', $errorData);
+				$phpcsFile->addWarning($error, $stackPtr, 'PublicWithUnderscore', $errorData);
 				return;
```

Comment:
I think there's a better way to address these in the ruleset.xml file instead of having to hand-edit the actual sniffs themselves. Unless I'm mistaking, you can change the severity of individual sniffs in the ruleset.xml file, essentially leaving the PHP code alone and "downgrading" the `ERROR` into a `WARNING` after the fact.

This commit (26535b8) is an example where I realized this myself and can be used for reference.